src/main/java/ledger/database/storage/ISQLiteDatabaseAccount.java

```diff
@@ -62,15 +66,15 @@ default void editAccount(Account account) throws StorageException {
     default List<Account> getAllAccounts() throws StorageException {
         try {
             Statement stmt = getDatabase().createStatement();
-            ResultSet rs = stmt.executeQuery("SELECT * FROM ACCOUNT;");
+            ResultSet rs = stmt.executeQuery("SELECT * FROM " + AccountTable.TABLE_NAME + ";");
```

Comment:
I got rid of all occurrences of "SELECT *" in the ISQLiteDatabase____ files in my refactoring work on the f_2refactoring branch.